test/testmisc.jl

```diff
@@ -0,0 +1,16 @@
+using Compat.Test, PyCall
+
+PyCall.@npyinitialize # see #481
+
+@testset "pair call syntax" begin
+    np = pyimport("numpy")
```

Comment:
Use `pybuiltin("list")` rather than have this test depend on `numpy` being installed.